kolibri/core/content/contentschema/versions/content_schema_current.py

```diff
@@ -47,34 +43,17 @@ class ContentLocalfile(Base):
 class ContentContentnode(Base):
     __tablename__ = "content_contentnode"
     __table_args__ = (
-        CheckConstraint("lft >= 0"),
-        CheckConstraint("tree_id >= 0"),
-        CheckConstraint("level >= 0"),
-        CheckConstraint("duration >= 0"),
-        CheckConstraint("rght >= 0"),
-        ForeignKeyConstraint(
-            ["lang_id"],
-            ["content_language.id"],
-            deferrable=True,
-            initially="DEFERRED",
-        ),
-        ForeignKeyConstraint(
-            ["parent_id"],
-            ["content_contentnode.id"],
-            deferrable=True,
-            initially="DEFERRED",
-        ),
```

Comment:
I don't know why this was deleted

Reply:
Because they should never have been added in the first place - adding these constraints into the sqlalchemy schema adds additional overhead to our channel import code, and is just a consequence of Django 3.2 doing foreign key integrity checks on SQLite now.

They also break tests when additional things are added to the schema.